test/suites/smoke-test-common/test-configuration-consistency.ts

```diff
@@ -72,12 +73,34 @@ describeSuite({
                 if (
                     Object.keys(authorities["orchestratorChain"]).length > config["minOrchestratorCollators"].toNumber()
                 ) {
+                    let containersToCompareAgainst: Vec<u32>;
+                    // If pending para ids for the session are empty we compare with registered para id, otherwise
+                    // we compare with pending para ids.
                     const liveContainers = await api.query.registrar.registeredParaIds();
+                    const pendingContainers = await api.query.registrar.pendingParaIds();
 
-                    expect(Object.keys(authorities["containerChains"]).length).to.be.equal(liveContainers.length);
+                    if (pendingContainers.length == 0) {
+                        containersToCompareAgainst = liveContainers;
+                    } else {
+                        for (let i = 0; i < pendingContainers.length; i++) {
+                            const entry = pendingContainers[i];
+                            if (entry[0].toNumber() == sessionIndex + 1) {
+                                containersToCompareAgainst = entry[1];
+                                break;
+                            }
+                        }
```

Comment:
Not sure if `.find()` works, if it doesn't try `pendingContainers.toJSON().find(`

```suggestion
const foundEntry = pendingContainers.find(entry => entry[0].toNumber() === sessionIndex + 1);
if (foundEntry) {
    containersToCompareAgainst = foundEntry[1];
}
```